docs/about/whats-new.ipynb

```diff
@@ -774,8 +774,8 @@
     "        'y':sc.Variable(dims=['time'], unit='m', values=np.random.rand(N)),\n",
     "        'time':sc.Variable(dims=['time'], values=(10000*np.random.rand(N)).astype('datetime64[s]')),\n",
     "    })\n",
-    "binned = sc.bin(data, edges=[sc.linspace(dim='x', unit='m', start=0.0, stop=1.0, num=5),\n",
-    "                             sc.linspace(dim='y', unit='m', start=0.0, stop=1.0, num=5)])\n",
+    "binned = sc.bin(data, bins=[sc.linspace(dim='x', unit='m', start=0.0, stop=1.0, num=5),\n",
```

Comment:
To make it consistent with `histogram`.
I thought while I am breaking the API I might as well get this one out of the way. Would you rather postpone it?

Reply:
My point is: we should have a discussion first before making API changes.

What if we actually prefer to call it `edges` every rather than `bins`? We don't want to break things twice.

Reply:
Ok